File: app/src/main/java/com/emiryanvl/tapreader/data/remote/responses/OpenLibrarySearchApiResponse.java
package com.emiryanvl.tapreader.data.remote.responses;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class OpenLibrarySearchApiResponse {

    @SerializedName("docs")
    List<Doc> docs;

    public OpenLibrarySearchApiResponse(List<Doc> docs) {
        this.docs = docs;
    }

    public List<Doc> getDocs() {
        return docs;
    }

    public static class Doc {

        @SerializedName("title")
        String title;

        public Doc(String title) {
            this.title = title;
        }

        public String getTitle() {
            return title;
        }
    }
}

File: app/src/main/java/com/emiryanvl/tapreader/data/remote/responses/openLibrary/OpenLibrarySearchApiResponse.kt
package com.emiryanvl.tapreader.data.remote.responses.openLibrary

import com.google.gson.annotations.SerializedName

data class OpenLibrarySearchApiResponse(
    @SerializedName("docs")
    val docs: List<Doc>
)

data class Doc(
    @SerializedName("title")
    val title: String
)
